docs/articles/documentation/reference/test-api/testcontroller/getcurrentwindow.md

```diff
@@ -4,12 +4,12 @@ title: t.getCurrentWindow Method
 permalink: /documentation/reference/test-api/testcontroller/getcurrentwindow.html
 ---
 
-# t.getCurrentWindow method
+# t.getCurrentWindow Method
 
-Retrieves a window descriptor that corresponds to the active window.
+Retrieves a window descriptor that corresponds to the active window. Can be chained with other `TestController` methods.
 
-```js
-t.getCurrentWindow();
+```text
+t.getCurrentWindow() → Object
```

Comment:
Seems like `Promise<Object>` according to the example below

Reply:
it actually returns a WindowDescriptorPromise, a union type:
`this | Promise<unknown>`
Source: [1](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/test-controller.d.ts#L320) [2](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/test-controller.d.ts#L420) [3](https://github.com/DevExpress/testcafe/blob/afb1e857987cb2623c23529237716b34fb42bb5c/ts-defs-src/test-api/test-controller.d.ts#L85)